components/Card.js

```diff
@@ -16,31 +17,62 @@ export default function Card({
   return (
     <CardElement>
       {pathname === "/" && (
-        <DeleteButton aria-label="delete data" onClick={() => handleDelete(id)}>
+        <DeleteButton
+          aria-label="delete data"
+          onClick={() => handleDelete(id, "", "")}
```

Comment:
It's a little cleaner if you call the method with one argument only `handleDelete(id)` and give the method default values for the second and third parameters.

Reply:
done